### ISSUE TITLE
The focus order on the "Extension- details" screen is incorrect; it currently moves from right to left, going from the "Extension Resource" section to the "Extension Overview".: A11y_SQL Azure Data Studio_Extension_Keyboard

### ISSUE THREAD

**Issue by Mana-swi** [microsoft/azuredatastudio#26422]
### Tags:
#A11yTCS;#A11yWCAG;#WCAG2.4.3;#Focus Order;#Keyboard;#A11ySev2;#Win32;#DesktopApp;#BM_SQL Azure Data Studio_Win32_Sep25;#SQL Azure Data Studio;
**"[[Check out Accessibility Insights!]](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)) - Identify accessibility bugs before check-in and make bug fixing faster and easier."**
### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.52.0 (system setup)
VS Code: 1.82.0
Electron: 30.5.1
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22621
### Repro Steps:
1. Launch Azure Data Studio.
2. From the left menu, click the 'Extensions' icon to open the Extension Manager.
3. Choose any extension from the library.
4. The extension details screen will appear, tab through the screen and observe the focus shifts.
5. Observe that The focus order on the "Extension- details" screen is incorrect; it currently moves from right to left, going from the "Extension Resource" section to the "Extension Overview".
### Actual:
The focus order on the "Extension- details" screen is incorrect; it currently moves from right to left, going from the "Extension Resource" section to the "Extension Overview".

### Expected:
The focus order on the "Extension- details" screen should be left to right and top to bottom; it should move from the "Extension Overview" section to the "Extension Resource".

### User Impact:
Keyboard users may be misled about the content’s structure and hierarchy, which can result in misunderstanding the context, especially if they skip the overview at first.
### Recommendations:
 [MAS Problems and Suggested Fixes](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fmicrosoft.sharepoint.com%2fsites%2faccessibility%2fLists%2fMASProblemsAndSuggestedFixes%2fReviewed%2520Items.aspx&data=04%7c01%7cv-prajadhav%40microsoft.com%7cf642cb2fc53d4c4be38408d9d97cf216%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637779952946112692%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000&sdata=avswAOFlXEiWy%2fRT%2f%2fPGnurQzMGOUdSW7Z5Xp8ZioiQ%3D&reserved=0)
### MAS Reference:

- [MAS 2.4.3: Focus Order Guide](https://microsoft.sharepoint.com/sites/accessibility/SitePages/MAS-2.4.3-Focus-Order-Guide.aspx)
- [MAS 2.4.3 – Focus Order](https://aka.ms/MAS2.4.3)

![Image](https://github.com/user-attachments/assets/4a5311f9-2aea-4564-9da1-e22dc21ba136)
